hotel-management/src/routes/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Routes, Route } from 'react-router-dom';
+
+import ROUTE_PATH from '@/constants/routePath';
+import {
+  HomePage,
+  CartPage,
+  EntryPage,
+  RoomDetailsPage,
+  RoomFormPage,
+  Layout,
+  ErrorPage,
+} from '@/pages/index';
+
+const AppRoutes = () => {
+  const {
+    HOME_PAGE,
+    CART_PAGE,
+    LOGIN_PAGE,
+    REGISTER_PAGE,
+    ROOM_DETAILS_PAGE,
+    ADD_ROOM_PAGE,
+    EDIT_ROOM_PAGE,
+    ERROR_PAGE,
+  } = ROUTE_PATH;
+
+  return (
+    <Routes>
+      <Route element={<Layout />}>
+        <Route path={HOME_PAGE} element={<HomePage />} />
+        <Route path={CART_PAGE} element={<CartPage />} />
+        <Route path={LOGIN_PAGE} element={<EntryPage />} />
+        <Route path={REGISTER_PAGE} element={<EntryPage />} />
+        <Route path={ADD_ROOM_PAGE} element={<RoomFormPage action="ADD" />} />
+        <Route path={EDIT_ROOM_PAGE} element={<RoomFormPage action="EDIT" />} />
+        <Route path={ROOM_DETAILS_PAGE} element={<RoomDetailsPage />} />
+      </Route>
+      <Route path={ERROR_PAGE} element={<ErrorPage />} />
```

Comment:
Why we need this path

Reply:
What happen if I access to any route that u haven't defined yet

Reply:
The reason I defined the error route is to set up a route that captures all undefined paths and redirects users to an error page.